setup.py

```diff
@@ -15,12 +15,6 @@
 ##############################################################################
 """Setup for z3c.recipe.tag"""
 
-# Check Python version
-import sys
-if sys.version_info < (2, 6):
-    sys.exit("%s: need Python 2.6 or later.\nYour python is %s."
-             % (sys.argv[0], sys.version))
-
```

Comment:
Maybe it's time to start adding python_requires=... instead of this kind of thing?

Reply:
To be done together with meta/config.